Sources/SwiftProtobuf/FieldTypes.swift

```diff
@@ -50,6 +50,17 @@ public protocol FieldType {
 /// Marker protocol for types that can be used as map keys
 ///
 public protocol MapKeyType: FieldType {
+    /// A comparision function for where order is needed.  Can't use `Comparable`
+    /// because `Bool` doesn't conform, and since it is `public` there is no way
+    /// to add a conformance internal to SwiftProtobuf.
+    static func lessThan(lhs: BaseType, rhs: BaseType) -> Bool
```

Comment:
We could make the name of the method `_lessThan` to make it clear that it's only meant to be used by the library.

(We probably should have done that for the protocol itself, in retrospect.)

Reply:
Some of these actually leak into `Visitor` interface, so they sorta have to be `public` to allow someone to write a custom `Vistor`.

Prefixed this one.